js/setpassword.js

```diff
@@ -0,0 +1,52 @@
+OC.SetPassword = {
+	init : function() {
+		$('#set-password #submit').click(OC.SetPassword.setPassword);
+	},
+
+	setPassword : function(event){
+		event.preventDefault();
+		if ($('#password').val()){
+			$.post(
+					$('#password').parents('form').attr('action'),
+					{
+						password : $('#password').val()
+					},
+					OC.SetPassword.resetDone
+			);
+		}
+	},
+
+	resetDone : function(result){
```

Comment:
if this is not intended to be public, prefix the name with an underscore: `_resetDone`

Reply:
Moved to `_resetDone`.